internal/gcs/container.go

```diff
@@ -167,6 +168,8 @@ func (c *Container) Shutdown(ctx context.Context) (err error) {
 	defer func() { oc.SetSpanStatus(span, err) }()
 	span.AddAttributes(trace.StringAttribute("cid", c.id))
 
+	ctx, cancel := context.WithTimeout(ctx, 30*time.Second)
```

Comment:
For clarity `Shutdown` and `Terminate` are async events. So this is 30 seconds to receive the `Shutdown` / `Terminate` response. It does not indicate that the shutdown completed. We still have to `Wait()` on the container.

Reply:
Why not make this change in the caller, or in `shutdown()`?

Alternatively, should we have changed the global timeout on the bridge, either in the package or in the caller to the package? We already have a mechanism to terminate the bridge entirely if a synchronous RPC does not return in a certain time.